podman-image/Containerfile

```diff
@@ -43,6 +44,17 @@ RUN if [[ ${PODMAN_PR_NUM} == "" ]]; then \
         --from repo="copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next" \
         aardvark-dns crun netavark podman containers-common containers-common-extra crun-wasm; \
     else \
+        dnf -y install koji python3-koji-cli-plugins && \
+        # TODO: container-selinux installation from local rpm conflicts over selinux-policy.
+        koji download-build --latestfrom=${FEDORA_LATEST_STABLE}-updates-testing -a $(arch) crun || true &&\
+        for pkg in aardvark-dns containers-common netavark; \
+            do for tag in $(koji list-sidetags --user=packit | grep ${FEDORA_LATEST_STABLE}); \
+                do koji download-build --latestfrom=$tag -a $(arch) -a noarch $pkg || true; \
+            done; \
+        done && \
+        rm -f crun-krun*.rpm crun-wasm*.rpm && \
+        rpm-ostree override replace -C --experimental --freeze $(ls $(pwd)/*.rpm) || true && \
```

Comment:
here as as well, it means we never notice any problems with this command. Why do we ignore all errors?

Reply:
changed PTAL